Python/Github_Bomb_Issues/bomb-issues.py

```diff
@@ -0,0 +1,14 @@
+from github import Github
+import os
+import getpass
+
+username = input("Enter your username: ")
+password = getpass.getpass("Enter your Github password: ")
+g = Github(username, password)
+
+repository = input("Give the GitHub repository name that you want to bomb issues on (Example: rohankumawat/Github_automation): ")
+number = int(input("Give the number of issues that you want to bomb: "))
+repo = g.get_repo(repository)
+
+for x in range(number):
+    repo.create_issue(title="Lorem ipsum", body="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")
```

Comment:
Instead of hard-coding it, you can possibly make use of the RandomText API to get the spaghetti text and specify the same as the title and the Issue Body

Reply:
Okay! 
I'll give it a try.